### PR TITLE
Feature/test tool url ga4 gh

### PR DESCRIPTION
**Pull request by garyluu**
Display checker boolean and string in the Tools object.  Use TRS 2.0.0-beta.1

For #1145

### CODE REVIEW

File: dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java [hunk 1]
```diff
@@ -326,6 +328,21 @@ private static String baseURL(DockstoreWebserviceConfiguration config) throws UR
         return uri.toString();
     }
 
+    /**
+     * Gets the checker workflow GA4GH path (test_tool_path) if it exists
+     * @param config    The dockstore configuration file in order to find the base GA4GH path
+     * @param entry     The entry to find its checker workflow path (test_tool_path)
+     * @return
+     */
+    public static String getCheckerWorkflowPath(DockstoreWebserviceConfiguration config, Entry entry) {
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: Probably shouldn't be used anywhere else, make this private

File: dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java [hunk 2]
```diff
@@ -454,4 +471,20 @@ static ToolTests sourceFileToToolTests(SourceFile sourceFile) {
         toolTests.setTest(sourceFile.getContent());
         return toolTests;
     }
+
+    private static String getUrlFromId(DockstoreWebserviceConfiguration config, String newID) {
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: This too

Comment (garyluu) [reply dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: what to change for this?

File: dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java [hunk 3]
```diff
@@ -148,23 +148,25 @@ private static String extractHTTPPrefix(String gitUrl, String reference, String
...
         }
         // TODO: hook this up to a type field in our DB?
         io.swagger.model.Tool tool = new io.swagger.model.Tool();
         tool = setGeneralToolInfo(tool, container);
         tool.setId(newID);
         tool.setUrl(url);
-
+        String checkerWorkflowPath = getCheckerWorkflowPath(config, container);
+        if (checkerWorkflowPath == null) {
+            checkerWorkflowPath = "";
```
Comment (agduncan94) [inline dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: Is there a reason we don't do null?

Comment (garyluu) [reply dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: Swagger specification doesn't like it.  In fact, people defined x-nullable into the Swagger specifications to indicate that the properties are nullable

Comment (denis-yuen) [reply dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java]: Yeah, I think the cwl people ran into this with their python validator too

**Pull request merged**